gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Sat Jan 08 19:45:49 CST 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
```

Comment:
Should this version be the same as in `build.gradle`? I see 7.2.0 there, but maybe not really important.

Reply:
Where exactly in ```build.gradle```. I have looked, I don't find it. Can you provide a reference please. I think it may be neccessary.

Reply:
`distributionUrl` represents `Gradle` version
`com.android.tools.build:gradle` represents `Android Gradle plugin` version
Gradle is a bit chaotic

Reply:
@mdrlzy so all is good here?

Reply:
yep